Sources/ArgumentParser/Parsable Properties/Option.swift

```diff
@@ -239,6 +239,129 @@ public struct ArrayParsingStrategy: Hashable {
 
 extension ArrayParsingStrategy: Sendable { }
 
+// TODO: to format and create separate file
```

Comment:
thought: I agree that this is probably best considering how the code base is structured.